src/main/java/com/powsybl/nad/layout/LayoutParameters.java

```diff
@@ -6,19 +6,36 @@
  */
 package com.powsybl.nad.layout;
 
+import com.powsybl.nad.model.Point;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * @author Florian Dupuy <florian.dupuy at rte-france.com>
  */
 public class LayoutParameters {
     private boolean textNodesForceLayout = false;
     private double springRepulsionFactorForceLayout = 0.0;
+    private Map<String, Point> initialPositions = Collections.emptyMap();
 
     public LayoutParameters() {
     }
 
     public LayoutParameters(LayoutParameters other) {
         this.textNodesForceLayout = other.textNodesForceLayout;
         this.springRepulsionFactorForceLayout = other.springRepulsionFactorForceLayout;
+        this.initialPositions = new HashMap<>(initialPositions);
+    }
+
+    public Map<String, Point> getInitialPositions() {
+        return Collections.unmodifiableMap(initialPositions);
+    }
+
+    public LayoutParameters setInitialPositions(Map<String, Point> initialPositions) {
```

Comment:
Initial positions are closer to an input than to a parameter

Reply:
Applied your suggestion and moved it to the generic `Layout` API (these initial positions could be respected by any layout algorithm).